files/fr/web/css/zoom/index.md

```diff
@@ -11,7 +11,7 @@ translation_of: Web/CSS/zoom
 
 {{CSSRef}}{{Non-standard_header}}
 
-La propriété non-standard **`zoom`** permet de manipuler l'effet d'agrandissement d'un élément. Plutôt que cette propriété non-standard, mieux vaudra utiliser [les transformations CSS](/fr/docs/Web/CSS/CSS_Transforms/Using_CSS_transforms), si possible. Cependant, contrairement aux transformations CSS, **`zoom`** affecte la taille de l'élément.
+La propriété non-standard `zoom` permet de manipuler l'effet d'agrandissement d'un élément. Plutôt que cette propriété non-standard, mieux vaudra utiliser [les transformations CSS](/fr/docs/Web/CSS/CSS_Transforms/Using_CSS_transforms), si possible. Cependant, contrairement aux transformations CSS, **`zoom`** affecte la taille de l'élément.
```

Comment:
Similar case here (cf. [the en-US version](https://github.com/mdn/content/pull/19885/files#diff-dcb157bae093dc0f2097fdb977142d5babfb78b95cfee5932251c932a3489dc0R14))

```suggestion
La propriété non-standard **`zoom`** permet de manipuler l'effet d'agrandissement d'un élément. Plutôt que cette propriété non-standard, mieux vaudra utiliser [les transformations CSS](/fr/docs/Web/CSS/CSS_Transforms/Using_CSS_transforms), si possible. Cependant, contrairement aux transformations CSS, `zoom` affecte la taille de l'élément.
```